cmd/soci-snapshotter-grpc/main.go

```diff
@@ -268,6 +268,8 @@ func serve(ctx context.Context, rpc *grpc.Server, addr string, rs snapshots.Snap
 		}
 	}()
 
+	log.G(ctx).Infof("soci-snapshotter-grpc successfully started, listening on %q", addr)
```

Comment:
nit: Can we we use `WithField("address", addr)` instead of putting the address into the log messae?